indico/modules/designer/placeholders.py

```diff
@@ -407,3 +408,65 @@ class FixedImagePlaceholder(DesignerPlaceholder):
     def render(cls, item):
         buf = DesignerImageFile.get(item['image_id']).open()
         return Image.open(buf)
+
+
+class RegistrationFormFieldPlaceholder(DesignerPlaceholder):
```

Comment:
It would be nice if this also took the (somewhat new) picture field into account.

Usually this is checked (in various places, also e.g. in the designer to use a square-shaped item) by looking at the class' `is_image`, but of course we cannot do that here because this placeholder can be either an image or nor depending on the field...